test/test-applications/integrations/TestApplication.Log4NetBridge/Program.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using log4net.Config;
+using Microsoft.Extensions.Logging;
+
+[assembly: log4net.Config.XmlConfigurator(Watch = true)]
+
+namespace TestApplication.Log4NetBridge;
+
+internal static class Program
+{
+    private static readonly ActivitySource Source = new("TestApplication.Log4NetBridge");
+
+    private static void Main(string[] args)
+    {
+        if (args.Length == 2)
+        {
+            using var activity = Source.StartActivity("ManuallyStarted");
+
+            var logApiName = args[1];
+            if (logApiName == "log4net")
+            {
+                LogUsingLog4NetDirectly();
+            }
+            else if (logApiName == "ILogger")
+            {
+                LogUsingILogger();
+            }
```

Comment:
Add NotSupportedException or something like that when logapiname != log4net and ilogger.